speed.cpp

```diff
@@ -0,0 +1,23 @@
+#include "speed.h"
+
+Speed::Speed() : speed(25) {}
```

Comment:
Shouldn't the speed of the vehicle start at 0 mph, or whatever unit of speed, cause when we turn on the car it should be stationary.

Reply:
I did 25 so we can see if it is centered and how it would look like in normal racing conditions. Once we implement, it will be 0.